samples/unit-tests/chart/parallel-coordinates-updates/demo.js

```diff
@@ -145,3 +145,71 @@ QUnit.test('Update parallel coordinates plot', function (assert) {
         'After chart update parallelInfo.counter has correct value (#10081).'
     );
 });
+
+// Calculate yAxis extremes based on the range series points as well (#15752)
+QUnit.test('Take range series points into account while calculating yAxis extremes (#15752)', function (assert) {
+    const chart = Highcharts.chart('container', {
+        chart: {
+            parallelCoordinates: true
+        },
+        series: [{
+            type: 'arearange',
+            data: [
+                [23, 25],
+                [-1, -0.5],
+                [1.5, 4],
+                [1.5, 5],
+                [4, 5]
+            ]
+        }]
+    });
+
+    const firstAxisMin = chart.yAxis[0].dataMin,
+        firstAxisMax = chart.yAxis[0].dataMax,
+        thirdAxisMin = chart.yAxis[2].dataMin,
+        thirdAxisMax = chart.yAxis[2].dataMax;
+
+    assert.deepEqual(23, firstAxisMin, '1st yAxis min should be 23.');
+    assert.deepEqual(25, firstAxisMax, '1st yAxis max should be 25.');
+
+    assert.deepEqual(1.5, thirdAxisMin, '3rd yAxis min should be 1.5.');
+    assert.deepEqual(4, thirdAxisMax, '3rd yAxis max should be 4.');
```

Comment:
When comparing numbers, use `assert.strictEqual`.

Also, it's better to use universal values like:
```js
assert.strictEqual(
    series.points[0].low, // is it low? I don't remember but you'll know
    firstAxisMin,
    ....
);
```